### PR TITLE
react router installed, two basic pages set, decreased background size

### PR DESCRIPTION
**Pull request by borisolah**
Quick edit for the description:

Converted the png background file to webp. Installed react-router-dom and added a few basic routes and pages to test it out.

### CODE REVIEW

File: client/src/layouts/RootLayout.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Outlet, NavLink } from "react-router-dom";
+export default function RootLayout() {
+  return (
+    <div className="root-layout">
+      <header>
+        <nav>
+          <h1>Jobarouter</h1>
```
Comment (csillagkrisztian) [inline client/src/layouts/RootLayout.js:7]: What is a Jobarouter?

File: client/src/pages/Home/home.css [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.home {
```
Comment (csillagkrisztian) [inline client/src/pages/Home/home.css:1]: Why is there an empty css class?

File: client/src/layouts/RootLayout.js (hunk 1 above)
Comment (borisolah) [reply client/src/layouts/RootLayout.js:7]: Just wanted to write an h1 to see if it shows properly or if I need to change the CSS. 
Am going to need to work on the CSS next.

File: client/src/pages/Home/home.css (hunk 2 above)
Comment (borisolah) [reply client/src/pages/Home/home.css:1]: In preparation of the following CSS changes!

Comment (csillagkrisztian) [reply client/src/pages/Home/home.css:1]: Ideally the CSS changes should be a separate project, I will approve this time ,but please try to not introduce unrelated or unfinished work on master.

File: client/src/layouts/RootLayout.js (hunk 1 above)
Comment (csillagkrisztian) [reply client/src/layouts/RootLayout.js:7]: If it's a test, then remove it before adding it to master.

File: client/src/pages/Home/home.css (hunk 2 above)
Comment (borisolah) [reply client/src/pages/Home/home.css:1]: Makes perfect sense, thanks for making things clear. I will look to keep the master branch clean and only push content that is finished.

**Review by csillagkrisztian — APPROVED** [review]

**Pull request merged**